infrastructure/ansible/roles/ats/defaults/main.yml

```diff
@@ -14,7 +14,16 @@
 #
 
 install_ats: False
-ort_pkg_name: "{% if ort_version is defined and ort_version != omit %}{{ 'traffic_ops_ort-'+ort_version }}{% else %}traffic_ops_ort{% endif %}"
+ort_pkg_name: >
+  {% if ort_version is defined and ort_version != omit %}
+    {% if ort_version >= 6.0.0.11351 %}
```

Comment:
The version needs single quotes everywhere

Reply:
Done